StyleCop.Analyzers/StyleCop.Analyzers.Test/OrderingRules/SA1209UnitTests.cs

```diff
@@ -88,12 +118,28 @@ class A
     }
 }";
 
+            var fixedTestCode = @"namespace Test
+{
+    using global::System;
```

Comment:
:exclamation: A namespace prefixed with `global::` should not ever be considered a `System` using directive.